processing/src/main/java/org/apache/druid/frame/key/ByteRowKeyComparator.java

```diff
@@ -44,23 +45,30 @@ public class ByteRowKeyComparator implements Comparator<byte[]>
   private final int firstFieldPosition;
   private final RowKeyComparisonRunLengths rowKeyComparisonRunLengths;
   private final RowSignature rowSignature;
```

Comment:
I don't think the `rowSignature` field is needed any longer.

Reply:
Weird that it didn't get flagged by the IDE.